### PR TITLE
Update opensource.md

### PR DESCRIPTION
**Pull request by chapnitsky**
Added the repositories that I selected and run by docker containers.

### CODE REVIEW

File: docker/opensource.md [hunk 1]
```diff
@@ -1,6 +1,106 @@
...
+* [Dialogy](https://github.com/skit-ai/dialogy)
+
+Steps to run tests on a docker container:
+```
+git clone https://github.com/skit-ai/dialogy.git
+cd dialogy
+docker run -it --name dialogy -w /opt -v$(pwd):/opt python:3.11 bash
+pip install poetry
+poetry install
+make install
```
Comment (szabgab) [inline docker/opensource.md:28]: why not "make test" ?

File: docker/opensource.md [hunk 2]
```diff
@@ -1,6 +1,106 @@
 # Open Source
 {id: open-source}
 
+## TOML Kit
```
Comment (szabgab) [inline docker/opensource.md:4]: If you look at the other page in the file you'll see that under each page title there is also an id that will become the url of the page.

File: docker/opensource.md [hunk 3]
```diff
@@ -1,6 +1,106 @@
...
+## Teiphy
+* [Teiphy](https://github.com/jjmccollum/teiphy)
+
+Steps to run tests on a docker container:
+```
+git clone https://github.com/jjmccollum/teiphy.git
+cd teiphy
+docker run -it --name teiphy -w /opt -v$(pwd):/opt python:3.11 bash
+pip install poetry
+pip install pytest
```
Comment (szabgab) [inline docker/opensource.md:40]: there is no need to install pytest manually

File: docker/opensource.md [hunk 4]
```diff
@@ -1,6 +1,106 @@
...
+## Python Automation Framework
+* [python-automation-framework](https://github.com/mreiche/python-automation-framework)
+
+Steps to run tests on a docker container:
+```
+git clone https://github.com/mreiche/python-automation-framework.git
+cd python-automation-framework
+docker run -it --name python-automation-framework -w /opt -v$(pwd):/opt python:3.11 bash
+pip install pytest
+PYTHONPATH="." pytest --numprocesses=4 --cov=paf test
```
Comment (szabgab) [inline docker/opensource.md:54]: I get:

ImportError while loading conftest '/opt/test/conftest.py'.
test/__init__.py:4: in <module>
    import inject
E   ModuleNotFoundError: No module named 'inject'



Have you maybe forgotten to add

pip install -r requirements.txt

?

File: docker/opensource.md [hunk 5]
```diff
@@ -1,6 +1,106 @@
...
+## Python Bitcoinlib
+* [python-bitcoinlib](https://github.com/petertodd/python-bitcoinlib)
+
+Steps to run tests on a docker container:
+```
+git clone https://github.com/petertodd/python-bitcoinlib.git
+cd python-bitcoinlib
+docker run -it --name python-bitcoinlib -w /opt -v$(pwd):/opt python:3.11 bash
+python3 -m unittest discover
+```
```
Comment (szabgab) [inline docker/opensource.md:66]: This is ok

File: docker/opensource.md (hunk 4 above)
Comment (chapnitsky) [reply docker/opensource.md:54]: Yes.. I'll add it

**Review by chapnitsky — COMMENTED** [review]
I think I fixed it all

**Pull request merged**